packages/compass-indexes/src/components/search-index-template-dropdown/index.tsx

```diff
@@ -0,0 +1,74 @@
+import { ATLAS_SEARCH_TEMPLATES } from '@mongodb-js/mongodb-constants';
+import type { SearchTemplate } from '@mongodb-js/mongodb-constants';
+import React, { useState, useCallback } from 'react';
+import {
+  Select,
+  Option,
+  Icon,
+  css,
+  spacing,
+  Tooltip,
+} from '@mongodb-js/compass-components';
+
+const dropdownStyles = css({
+  display: 'flex',
+  overflow: 'hidden',
```

Comment:
This overflow hidden in combination with a margin hack for the icon makes the right border of the input to disappear. You generally only want to use overflow when you intentionally want to not allow scrolling and hide stuff that is out of the element box